tests/FrameworkTests/FrameworkTests.csproj

```diff
@@ -50,28 +50,28 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="Microsoft.Extensions.Configuration.Abstractions, Version=2.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60, processorArchitecture=MSIL">
-      <HintPath>..\packages\Microsoft.Extensions.Configuration.Abstractions.2.0.0\lib\netstandard2.0\Microsoft.Extensions.Configuration.Abstractions.dll</HintPath>
+      <HintPath>..\..\packages\Microsoft.Extensions.Configuration.Abstractions.2.0.0\lib\netstandard2.0\Microsoft.Extensions.Configuration.Abstractions.dll</HintPath>

```

Comment:
Any idea why the hint paths got changed?

Reply:
I manually edited this because restore put the packages in the root dir which is two folder up, in order to compile on a clean check out. Happy to leave this out of the PR if needed.

Reply:
Nope - leave it. Was just wondering what happened. I’d much rather it compile from check-out, not sure why my setup was different.